.github/workflows/tests-cron.yml

```diff
@@ -0,0 +1,44 @@
+name: Tests Cron
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9 * * 1-5" # At 09:00 AM, Monday through Friday
+
+jobs:
+  tests:
+    name: Long running Tests
+    runs-on: ubuntu-latest
+    continue-on-error: true # we want the issue to popup even if sth goes wrong
```

Comment:
why do you need this?
You already added the if condition to manage the case of failure. https://github.com/Talend/ui/pull/3871/files#diff-d3782acde429bd2a63f81d47345c3de6f798bad3d04f757d2813bb77ece283a4R32

Reply:
indeed, let's remove it